flusher.go

```diff
@@ -306,7 +320,7 @@ func (s *Server) flushPart(metricSlice []samplers.DDMetric, wg *sync.WaitGroup)
 	}, "flush", true)
 }
 
-func (s *Server) flushForward(wms []WorkerMetrics) {
+func (s *Server) flushForward(wms []WorkerMetrics, interval time.Duration) {
```

Comment:
Looks like we're no longer using the `interval` here. Was this left over from your previous rate-based work?